File: android/app/src/main/java/com/choicecrafter/students/models/NotificationType.java
package com.choicecrafter.studentapp.models;

public enum NotificationType {
    ACTIVITY_STARTED("Activity Started"),
    COLLEAGUE_ACTIVITY_STARTED("Colleague Activity Started"),
    POINTS_THRESHOLD_REACHED("Points Milestone"),
    COMMENT_ADDED("Comment Added"),
    CHAT_MESSAGE("Chat Message"),
    REMINDER("Reminder");

    private final String displayName;

    NotificationType(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}

File: lib/models/notification_type.dart
enum NotificationType {
  activityStarted('Activity Started'),
  colleagueActivityStarted('Colleague Activity Started'),
  pointsThresholdReached('Points Milestone'),
  commentAdded('Comment Added'),
  chatMessage('Chat Message'),
  reminder('Reminder');

  const NotificationType(this.displayName);

  final String displayName;
}
